package.json

```diff
@@ -42,9 +43,11 @@
     "pug-loader": "^2.4.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "responsive-loader": "^2.3.0",
```

Comment:
Are you using all these?

Reply:
A bunch of those were 'needed' when I was trying to get images to work and didn't know how file routing worked. Probably don't need any of the loaders anymore.

Reply:
I'll reset package.json to the one on main branch